.travis.yml

```diff
@@ -32,15 +36,9 @@ jobs:
     - language: php
       php: 7.2
       stage: test
-    - language: php
-      php: 7.1
-      stage: test
     - language: php
       php: 7.0
       stage: test
-    - language: php
-      php: 5.6
```

Comment:
PHP 5.6 is provided in CentOS 7, which we currently support. We need to have the discussion about continuing that support, but I'll open an issue for that

Reply:
#1113 1113